seqr/views/utils/permissions_utils.py

```diff
@@ -19,14 +22,33 @@ def get_project_and_check_permissions(project_guid, user, **kwargs):
     return project
 
 
+def project_has_anvil(project):
+    return anvil_enabled() and bool(project.workspace_namespace and project.workspace_name)
+
+
+def anvil_has_perm(user, permission_level, project):
+    if not anvil_enabled():
+        return False
+    collaborators = sa_get_workspace_acl(project.workspace_namespace, project.workspace_name) if project_has_anvil(project) else {}
+    if user.email in collaborators.keys():
+        permission = collaborators[user.email]
+        if permission['pending']:
+            return False
+        if permission_level in [CAN_EDIT, IS_OWNER]:
+            return (permission['accessLevel'] in ['WRITER', 'OWNER'])
```

Comment:
I merged `CAN_EDIT` and `IS_OWNER` permissions because there are no tests for `IS_OWNER` and I heard from you that these two permissions literally the same. Can they be merged here?

Reply:
yes this seems fine to me